src/interp.cc

```diff
@@ -1358,6 +1360,24 @@ ValueTypeRep<T> SimdSplat(V lane_data) {
   return ToRep(Bitcast<T>(simd_data));
 }
 
+// Simd instructions with Lane operand.
+// a0: input v128 value. a1: lane index. 
+// typename T: lane data type.
+template <typename R, typename V0, typename V1, typename T>
```

Comment:
Is there a need to have differing lane index types? I think this can always be int (instead of using V1).

Reply:
Agree.

Reply:
Done.